packages/storage/README.md

```diff
@@ -0,0 +1,11 @@
+# storage
```

Comment:
Can you please update the readme to be a bit more detailed, compared to boilerplate one.

Reply:
Sure. Give me some time to update it